js/admin.js

```diff
@@ -0,0 +1,1019 @@
+        email: "admin@gmail.com",
+        password: newPassword
+    }
+    let jsonAdminAcount = JSON.stringify(adminAccountObject);
+    localStorage.setItem('adminAccount', jsonAdminAcount);
+    adminAccount = adminAccountObject;
+    console.log(adminAccount);
+
+    // Add Notification
+
+}
+///////////////////////////////////// END Handling admin reset password /////////////////////////////////////
+
+
+
+///////////////////////////////////// START Handling admin sidebar /////////////////////////////////////
+
+// Hanle Click on category To show sub category in Admin Dashboard (sidebar)
+let categoryListContainer = document.querySelector('#category-list-container');
+if (categoryListContainer) {
+    categoryListContainer.addEventListener('click', _ => {
+        let categoryList = document.querySelector('#category-list');
+        categoryList.classList.toggle('hide');
+
+        // Change Arrow status
+        let categoryListArrow = categoryListContainer.querySelector('span > i');
+        if (categoryListArrow.classList.contains('fa-angle-right'))
+            categoryListArrow.classList.replace('fa-angle-right', 'fa-angle-down');
+        else if (categoryListArrow.classList.contains('fa-angle-down'))
+            categoryListArrow.classList.replace('fa-angle-down', 'fa-angle-right');
+
+    });
+}
+
+// Hanle Click on product To show sub product in Admin Dashboard (sidebar)
+let productListContainer = document.querySelector('#product-list-container');
+if (productListContainer) {
+    productListContainer.addEventListener('click', _ => {
+        let productList = document.querySelector('#product-list');
+        productList.classList.toggle('hide');
+
+        // Change Arrow status
+        let productListArrow = productListContainer.querySelector('span > i');
+        if (productListArrow.classList.contains('fa-angle-right'))
+            productListArrow.classList.replace('fa-angle-right', 'fa-angle-down');
+        else if (productListArrow.classList.contains('fa-angle-down'))
+            productListArrow.classList.replace('fa-angle-down', 'fa-angle-right');
+    });
+}
+///////////////////////////////////// END Handling admin sidebar /////////////////////////////////////
+
+
+///////////////////////////////////// START Handling admin category /////////////////////////////////////
+
+// Add Category
+let addCategorybtn = document.querySelector('#admin-add-category .btn');
+if (addCategorybtn) {
+    addCategorybtn.addEventListener('click', addCategory);
+}
+
+// Search in category 
+function findCategory(categoryName, oldCategories, id = 0) {
+    for (let index = 0; index < oldCategories.length; index++) {
+        if (oldCategories[index].name === categoryName && oldCategories[index].id != id) {
+            return true;
+        }
+    }
+    return false;
+
+    // oldCategories.forEach(element => {
+    //     if(element.name === categoryName){
+    //         return true;
+    //     }
+    // });
+    // return false;
```

Comment:
Did you forget to uncomment those lines, or they were for testing? If those lines are for testing your code please remove them.

Reply:
No, I wanna ask our instructor why this block of code didn't work correctly.
but you have a right, in the final version all unuseful comments must be removed.
Do you do it? or I update it before you merge

Reply:
Remove before merge

Reply:
Done